crypto/signer.go

```diff
@@ -31,39 +31,39 @@ var sigpbHashLookup = map[crypto.Hash]sigpb.DigitallySigned_HashAlgorithm{
 // Signer is responsible for signing log-related data and producing the appropriate
 // application specific signature objects.
 type Signer struct {
-	hash   crypto.Hash
-	signer crypto.Signer
+	Hash   crypto.Hash
+	Signer crypto.Signer
 }
 
 // NewSigner creates a new Signer wrapping up a hasher and a signer. For the moment
 // we only support SHA256 hashing and either ECDSA or RSA signing but this is not enforced
 // here.
 func NewSigner(signer crypto.Signer) *Signer {
```

Comment:
`NewSigner()` now seems a little odd -- either drop it, rename it to `NewDefaultSigner`, or add hash/signer params?